build/bazel/remote/execution/v0_2/remote_execution.proto

```diff
@@ -580,6 +581,39 @@ message Digest {
   int64 size_bytes = 2;
 }
 
+// ExecutedActionMetadata contains details about a completed execution.
+message ExecutedActionMetadata {
+  // The name of the worker which ran the execution.
+  string worker = 1;
+
+  // When was the action added to the queue.
+  google.protobuf.Timestamp queued_timestamp = 2;
```

Comment:
Would this make more sense as a key-value map? Not all systems may expose the same metrics. We could provide a standard list of keys if desired.

Reply:
These particular metrics seem general enough, though. Also, the map would be much heavier on the wire. How about we add a map for server-specific times later if/when we need it?